bigbluebutton-html5/imports/ui/components/logout-confirmation/container.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { createContainer } from 'meteor/react-meteor-data';
+import userListService from '../user-list/service';
+import LogoutConfirmation from './component';
+import LogoutConfirmationService from './service';
+
+class LogoutConfirmationContainer extends Component {
+    
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return(
+      <LogoutConfirmation {...this.props} />
+    );
+  }
+}
+
+export default createContainer(() => {
+
+  return {
+    isModerator: userListService.getCurrentUser().isModerator,
```

Comment:
Why is logout-confirmation using methods from the UserList service?

Reply:
Because this endMeeting button is only displayed for the moderator. So I use the method from the UserList service to check whether the current user is a moderator or not.

Reply:
This shouldn't happen. logout-confirmation has its own service.js.
Just copy-paste the method, if you wish. Probably we should collect such methods and place under one common service at some point, but one component shouldn't be using service methods from another.